src/main/java/nl/roboteamtwente/autoref/model/Robot.java

```diff
@@ -11,6 +11,11 @@ public class Robot extends Entity {
      */
     private final int id;
 
+    /**
+     * Maximum radius of the robot in meters
+     */
+    private final double radius = 0.09;
```

Comment:
Is there are reason that this variable is hard-coded? Some robots are wider than others (this information should be known by SSL-vision). 

In the case that you're okay with the error I recommend making it a proper constant:
```suggestion
    public static final double radius = 0.09;
```
This makes sure that you do not have to pass each robot object to all of the funtions